File: python/1551.py
n = int(input())

for i in range(n):
    x = input()
    x = x.replace(' ', '')
    x = x.replace(',', '')
    
    arr = []
    
    for g in range(len(x)):
        item = x[g]
        try:
            flag = arr.index(item)
        except:
            arr.append(item)
            
    force = len(arr)
    msg = ''
    
    if force < 13:
        msg = 'frase mal elaborada'
    elif force == 26:
        msg = 'frase completa'
    else:
        msg = 'frase quase completa'
        
    print(msg)

File: lua/1551.lua
local n = tonumber(io.read())

for i=1,n do
    local phrase = io.read()
    local count = 0
    for j=97,122  do
        local char = string.char(j)
        local ind = string.find(phrase, char)
        if ind then
            count = count + 1
        end
    end
    if count == 26 then
        print('frase completa')
    elseif count >= 13 then
        print('frase quase completa')
    else
        print('frase mal elaborada')
    end
end